config/jobs/kubernetes/cluster-registry/cluster-registry-config.yaml

```diff
@@ -30,7 +30,7 @@ presubmits:
       preset-k8s-ssh: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180829-29e661965-1.8
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180829-29e661965-master
```

Comment:
this will get automatically upgraded to the current stable image we're using for kubernetes master branch testing as we roll it out in the future

Reply:
That's exactly what we want. Thank you, @BenTheElder!

Reply:
Great! You're welcome :smile:  
Your other jobs already appear to be tracking this. I did a quick check and only some generated kubernetes 1.8 jobs we need to trim out use this image.